Octokit.Reactive/Clients/IObservableInstallationsClient.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace Octokit.Reactive
+{
+    public interface IObservableGitHubAppsInstallationsClient
+    {
+        /// <summary>
+        /// List repositories of the authenticated GitHub App (requires GitHubApp JWT token auth).
```

Comment:
The docs says this endpoint uses an installation token, not a JWT... and this would get repositories of an installation rather than a github app (which would have many installations each with potentially many repos)

So let's go with "List repositories of the authenticated installation (requires Installation Token auth)" or something similar